bucket/imagemagick.json

```diff
@@ -19,7 +22,7 @@
         "conjure.exe",
         "convert.exe",
         "dcraw.exe",
-        "ffmpeg.exe",
+        ["ffmpeg.exe", "iffmpeg"],
```

Comment:
@rrelmy how does that sound to you? I will merge this since the current version is 404'ing, but please let us know or submit a pull request if there's a better way.